.github/workflows/release.yaml

```diff
@@ -0,0 +1,54 @@
+name: Release
+
+on:
+  workflow_call:
+    inputs:
+      config-name:
+        required: true
+        type: string
+    secrets:
+      AUTOMATION_TOKEN:
+        required: true
+  
+permissions:
+  contents: read
+
+jobs:
+  initial_release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+    - name: Git checkout
+      uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+    - name: checkout labs-tools-scripts repo
+      uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.AUTOMATION_TOKEN }}
+        repository: boldlink/labs-tools-scripts
+        ref: 'main'
+        path: .tmp      
+    - name: run release script
+      shell: bash
+      run: bash ${GITHUB_WORKSPACE}/.tmp/scripts/github_action/release.sh && rm -rf ${GITHUB_WORKSPACE}/.tmp/
```

Comment:
How is this file made available on the repo executing the workflow? Should we have a step to pull it from somewhere before?

Reply:
Previously we were using a python script to pull the bash script. In the current setup, we are checking out the repo in which the file is hosted, running it, then doing a cleanup after the script has been ran.

Reply:
Fair enough, how are you handling the priorities and order of running scripts?

Reply:
In the release workflow we are only running one script, and the workflow only runs when code is pushed to main.